index.html

```diff
@@ -15,7 +15,6 @@
     <p class="lead">Druid is a fast column-oriented distributed data store</p>
     <p><a class="btn btn-lg btn-outline-inverse" href="downloads.html"><span class="fa fa-download"></span> Download</a>
     <a class="btn btn-lg btn-outline-inverse" href="https://github.com/druid-io/druid/"><span class="fa fa-github"></span> GitHub</a></p>
-    <p class="small text-muted">latest stable release {{ site.druid_stable_version }}</p>
```

Comment:
also the download page includes the latest stable version. I don't see what including it so prominent on the front page does